doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -94,6 +94,14 @@ File level preview tools allow the user to see a preview of the file contents wi
 
 When a file has a preview available, a preview icon will appear next to that file in the file listing on the dataset page. On the file page itself, the preview will appear in a Preview tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
 
+File Level Query Tools
+++++++++++++++++++++++++
+
+File level query tools allow the user to ask natural language queries of data table's contents without having to download it.
+
+When a file has a query tool available, a query icon will appear next to that file in the file listing on the dataset page. On the file page itself, the query tool will appear in a Query tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
+
```

Comment:
One of these things is not like others:

![Screen Shot 2023-07-26 at 3 18 08 PM](https://github.com/IQSS/dataverse/assets/21006/c951547b-b388-4ab2-b001-de81683a0dd9)
![Screen Shot 2023-07-26 at 3 18 16 PM](https://github.com/IQSS/dataverse/assets/21006/9717498f-a9da-4fe4-b1dd-62d8fd8fb59f)
![Screen Shot 2023-07-26 at 3 18 22 PM](https://github.com/IQSS/dataverse/assets/21006/b81bb960-bfff-4296-b2a0-085f3fb3df63)
![Screen Shot 2023-07-26 at 3 18 26 PM](https://github.com/IQSS/dataverse/assets/21006/a878552b-e8dd-40e0-8dd0-d5a3becf1ddc)